src/app/components/quotes/quotes.component.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+<body>
+
+<div class="container">
```

Comment:
I think you can delete everything besides lines 5-8, since that's where the actual content is. Angular 2 is pretty awesome and it automatically imports the CSS and TypeScript needed, and we already have Open Sans imported ;)